### PR TITLE
DOCSP-29118 install mongosh with openssl

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-29118-install-mongosh-with-openssl/install/#std-label-mdb-shell-install)

[JIRA](https://jira.mongodb.org/browse/DOCSP-29118)
[SERVER][Docs Feedback] Clarify mongosh install instructions on .deb

[BUILD](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=64494d157efd2382b3bdcade)

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/includes/steps-install-shell-linux-deb.yaml [hunk 1]
```diff
@@ -93,25 +93,23 @@ level: 4
 ref: install
 content: |
 
-  To install the latest stable version of ``mongosh``, issue the
-  following command:
+  ``mongosh`` supports OpenSSL. You can install ``mongosh`` without
+  OpenSSL or you can use your system's OpenSSL installation to enhance
```
Comment (ajhuh-mdb) [inline source/includes/steps-install-shell-linux-deb.yaml:97]: Super small grammar nit

```suggestion
  OpenSSL, or you can use your system's OpenSSL installation to enhance
```

File: source/includes/steps-install-shell-linux-rpm.yaml [hunk 2]
```diff
@@ -74,25 +74,23 @@ level: 4
 ref: install
 content: |
 
-  To install the latest stable version of ``mongosh``, issue the
-  following command:
+  ``mongosh`` supports OpenSSL. You can install ``mongosh`` without
```
Comment (ajhuh-mdb) [inline source/includes/steps-install-shell-linux-rpm.yaml:77]: [suggestion] Could potentially put this blurb in an include given it's also used in the Debian instructions?

**Review by ajhuh-mdb — APPROVED** [review]
LGTM with some a couple non-blocking comments!

File: source/includes/steps-install-shell-linux-deb.yaml (hunk 1 above)
Comment (davemungo) [reply source/includes/steps-install-shell-linux-deb.yaml]: Done

File: source/includes/steps-install-shell-linux-rpm.yaml (hunk 2 above)
Comment (davemungo) [reply source/includes/steps-install-shell-linux-rpm.yaml]: Done

File: source/includes/steps-install-shell-linux-rpm.yaml [hunk 3]
```diff
@@ -74,25 +74,23 @@ level: 4
...
+  OpenSSL or you can use your system's OpenSSL installation to enhance
```
[the remark this answers is not in the document]
Comment (davemungo) [reply source/includes/steps-install-shell-linux-rpm.yaml]: Done

**Review by davemungo — COMMENTED** [review]
@ajhuh-mdb thanks for the feedback

**Comment by davemungo** [comment]
@addaleax thanks for the feedback. This is ready for another look when you have a moment.

**Review by addaleax — APPROVED** [review]

**Pull request merged**